### PR TITLE
Feature/dynamic routes

### PR DESCRIPTION
**Pull request by lbmerchant93**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Create dynamic routes when selecting a date from the calendar input.
Clean up methods in the Header component

## *Instructions for reviewer:*
*Where should the reviewer start?*
Head.vue line 4 
App.vue line 2

*How should this be manually tested?*
url should change with the date selected, all information should still display the same

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
The history in the browser does not work, can not use the back/forward arrows in browser.

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -35,6 +35,9 @@ export default {
   },
   methods: {
     showPicture(date) {
+      if(date === undefined) {
+        date = this.todaysDate;
```
Comment (colefiscus) [inline src/App.vue:39]: Sorry maybe I can't read, where are we getting this.todaysDate from?

File: src/App.vue [hunk 2]
```diff
@@ -46,7 +49,8 @@ export default {
         const nextDay = this.getNextDate(date);
         apiCalls.getSpecificDatesPhoto(nextDay)
           .then(photo => this.pond = photo)
-      }
+        this.onHome = false;
```
Comment (colefiscus) [inline src/App.vue:52]: Same question here

**Review by colefiscus — COMMENTED** [review]
Just had 2 small questions!

File: src/Header.vue [hunk 3]
```diff
@@ -10,43 +10,29 @@
...
 <script>
+import router from './router'
 export default {
   name: "Header",
+  props: ['onHome' , 'todaysDate'],
   data() {
     return {
       date: '',
-      today: this.findTodaysDate()
+      today: this.todaysDate
```
Comment (richardltyler) [inline src/Header.vue:29]: I like moving this method and value up to the `parent component`

**Review by richardltyler — APPROVED** [review]
Looks good !

**Pull request merged**